test/test_cli.py

```diff
@@ -516,3 +516,50 @@ def test_SBOM(self, caplog):
             logging.INFO,
             "There are 1 products with known CVEs detected",
         ) in caplog.record_tuples
+
+    def test_console_output_depending_reportlab_existence(self, caplog):
+        import sys
+        from importlib.machinery import ModuleSpec
+        from importlib.util import module_from_spec
+
+        my_test_filename = "reportlab_test_report.pdf"
```

Comment:
Can we make sure this test runs in the temp directory like we've done with other tests?  Sure, it's unlikely this filename will already be in use, but I'd rather just do it the right way.  You should be able to use the same construction you're using for the curl test file below.